### DISCUSSION TITLE
Pinging locally defined services

### DISCUSSION THREAD

**Discussion by vladfedoriuk** [hynek/svcs#81] [Ideas]
Hi!

Thanks for sharing your library, which has been incredibly useful in my projects.

I would like to ask a question - would you find it useful to make the `svcs._core.Container.get_pings` method include locally defined services with `svcs._core.Container.register_local_factory` or `svcs._core.Container.register_local_value` methods? Currently, it retrieves objects stored in the `registry` attribute of the `svcs._core.Container`, but I believe there might be a use-case for getting all the registered services defining a `ping` within the application.

Thanks in advance!

**Comment by hynek** [comment]
Hm yeah thanks that seems like an oversight! Tracked in #82.

**Comment by hynek** [comment]
fixed by #83
